source/overview/security.rst

```diff
@@ -2,74 +2,133 @@
 
-   - To protect against brute force attacks, you can set `rate limiting on APIs <https://docs.mattermost.com/administration/config-settings.html#id55>`_, varied by query frequency, memory store size, remote address and headers.
-   - Session length, session cache and idle timeout can be `configured according to your internal policies <https://docs.mattermost.com/administration/config-settings.html#sessions>`_, automatically forcing a user to re-login after a specified period of time.
-   - Remotely `revoke user sessions <https://docs.mattermost.com/help/settings/account-settings.html#view-and-logout-of-active-sessions>`_ across web, mobile devices and native desktop apps.
-   - Remotely reset user passwords via the System Console or via the `command line <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-password>`_.
-   - Mattermost supports integrated authentication with `Active Directory and LDAP <https://docs.mattermost.com/deployment/sso-ldap.html>`_ (E10) as well as `SAML 2.0 SSO integration <https://docs.mattermost.com/deployment/sso-saml.html>`_ with providers including `Active Directory Federation Services <https://docs.mattermost.com/deployment/sso-saml-adfs.html>`_,  `Okta <https://docs.mattermost.com/deployment/sso-saml-okta.html>`_, among others (E20).
-   - The ability to require `multi-factor authentication <https://docs.mattermost.com/deployment/auth.html>`_ is also available (E10).
+Users can set `rate limiting on APIs <https://docs.mattermost.com/administration/config-settings.html#id55>`_ to protect against brute force attacks varied by query frequency, 
+memory store size, 
+remote address,
+and headers.
+
+Session length, 
+session cache,
+and idle timeout can be `configured according to your internal policies <https://docs.mattermost.com/administration/config-settings.html#sessions>`_. This forces users to login again after a specified period of time.
+
+Remotely `revoke user sessions <https://docs.mattermost.com/help/settings/account-settings.html#view-and-logout-of-active-sessions>`_ across web, 
+mobile devices,
+and native desktop applications
+and remotely reset user passwords through the System Console or through the `command line <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-password>`_.
+
+Mattermost supports integrated authentication with `Active Directory and LDAP <https://docs.mattermost.com/deployment/sso-ldap.html>`_ on Mattermost Enterprise Editions E10 and E20,
+and `SAML 2.0 SSO integration <https://docs.mattermost.com/deployment/sso-saml.html>`_ with providers including `Active Directory Federation Services <https://docs.mattermost.com/deployment/sso-saml-adfs.html>`_,  `Okta <https://docs.mattermost.com/deployment/sso-saml-okta.html>`_, among others on Mattermost Enterprise Edition E20.
+
+Users can mandate `multi-factor authentication <https://docs.mattermost.com/deployment/auth.html>`_ on Mattermost Enterprise Editions E10 and E20.
```

Comment:
Maybe better to say "Administrators" or "System Administrators" rather than "Users"

Reply:
done